.cargo/config

```diff
@@ -1,10 +1,33 @@
-[target.thumbv7em-none-eabihf]
-runner = 'arm-none-eabi-gdb'
+[target.thumbv7m-none-eabi]
```

Comment:
This config was used in the project I was copying this code, from it did not break the automatic tests but if someone have a clue why please enlitghten me.

Reply:
I don't think this should be changed.
Why did you change this? Sorry, I don't understand from your explanation.

Reply:
Soo this is the part where I'm not sure what is happening and what is should be... which is a very unsatisfactory answer -- I copied this code form another project where it was working and this was one of the files I had to change.

With this change the new test case compiles and the old regression test are still building. @Yatekii  do you see what parts of this is needed and what should not be here?

Reply:
Well, the runner says what cargo run executes when built for said target. Also all the uncommented settings below will ofc. affect said target.
There is no build settings in there atm, so it didn't have any effect with travis.

The missing "hf" part means that it's only for non-hardware float.

So this should really be reverted back to what it was :)

I will have a look at the rest of the code later this evening. Looked good at first glance tho.